tests/python/frontend/mxnet/test_forward.py

```diff
@@ -505,6 +505,12 @@ def verify(xshape, yshape, y_data):
     verify((2, 2), (2, 3), [[1, 1, 0], [0, 1, 0]])
     verify((2, 2, 2), (2, 2), [[0, 1], [1, 0]])
 
+def test_forward_bilinear_resize():
+    # add tests including scale_height and scale_width when mxnet is updated to version 1.5
```

Comment:
@icemelon9 Please check if this looks ok.

Reply:
Already approved. Let's wait for CI to complete.